corehq/apps/users/views/__init__.py

```diff
@@ -512,7 +512,7 @@ def non_admin_roles(self):
         return list(sorted(
             [role for role in UserRole.objects.get_by_domain(self.domain) if not role.is_commcare_user_default],
             key=lambda role: role.name if role.name else '\uFFFF'
-        ))
```

Comment:
This will also make it available for web users, right?  I think we went back and forth on that but I don't recall for sure where we landed - that web users _should_ be able to be assigned the role too?

Reply:
This won't actually affect web users since the pages creating invitations and editing web users use a different code path.

Reply:
I added a commit to move these methods out of the base class into the only view where they are used: b5fb6922b28